components/custom-radio-button/css/template.js

```diff
@@ -0,0 +1,19 @@
+/**
+ * @license EUPL-1.2
+ * Copyright (c) 2021 Robbert Broersma
+ */
+
+import clsx from 'clsx';
+
+export const defaultArgs = {
+  checked: false,
+  disabled: false,
+};
+
+export const CustomRadioButton = ({ checked = false, disabled = false }) =>
+  `<input type="radio"${checked ? ' checked' : ''}${disabled ? ' disabled' : ''} class="${clsx(
+    'utrecht-custom-radio-button',
+    'utrecht-custom-radio-button--html-input',
+    checked && 'utrecht-custom-radio-button--checked',
+    disabled && 'utrecht-custom-radio-button--disabled',
```

Comment:
We should include all states here, such as the `active`, `focus`, `focus-visible` and `invalid` state, so we can display them in visual regression tests.